src/main/java/school/lemon/changerequest/java/Manager.java

```diff
@@ -0,0 +1,16 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 15.12.16.
+ */
+public class Manager extends Employee {
+
+    public Manager(String name) {
+        super(name);
+    }
+
+    @Override
+    public double calculateSalary() {
```

Comment:
It's exact copy of Accountant calculateSalary method - you should refactor your code to get rid of duplications.